app/config/SecureMessagingFeatureFlagConfig.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import features.Feature.Feature
+import features.SecureMessagingFeatureStatus.SecureMessagingFeatureStatus
+import features.{Feature, SecureMessagingFeatureStatus}
+import javax.inject.Inject
+import play.api.Configuration
+
+class SecureMessagingFeatureFlagConfig @Inject()(config: Configuration) {
```

Comment:
The class name seems a bit redundant. What about a simple _SecureMessagingConfig_? Even because it includes not only flags.

Reply:
Uhm... I see that we already have a _SecureMessagingConfig_ :confused:

Reply:
Yes, together with Glidester we named 2 classes the same way. Added a commit to rename this class again. Maybe the new name is more explanatory.